src/Text/Pandoc/Writers/XWiki.hs

```diff
@@ -122,6 +122,10 @@ blockToXWiki (DefinitionList items) = do
   contents <- local (\s -> s { listLevel = listLevel s <> ";" }) $ mapM definitionListItemToMediaWiki items
   return $ vcat contents <> if Text.null lev then "\n" else ""
 
+blockToXWiki (Figure attr _ body) = do
+  content <- blockToXWiki $ Div attr body
+  return $ intercalate content ["(((\n", ")))"]
+
```

Comment:
This creates a [group](https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/XWikiSyntax/?syntax=2.1&section=Groups)